R/pkg/R/DataFrame.R

```diff
@@ -2908,3 +2908,39 @@ setMethod("write.jdbc",
             write <- callJMethod(write, "mode", jmode)
             invisible(callJMethod(write, "jdbc", url, tableName, jprops))
           })
+
+#' randomSplit
+#'
+#' Return a list of randomly split dataframes with the provided weights.
```

Comment:
Could we explain that the dataframes are split by rows ? Its not really clear what randomSplit means without some context otherwise.